cross/libicu/Makefile

```diff
@@ -73,3 +75,7 @@ include ../../mk/spksrc.cross-cc.mk
 .PHONY: libicu_install_with_data_files
 libicu_install_with_data_files:
 	$(RUN) $(MAKE) DESTDIR=$(INSTALL_DIR) ICUPKGDATA_DIR=$(INSTALL_PREFIX)/share install PREFIX=$(INSTALL_PREFIX)
+
+.PHONY: libicu_post_install
+libicu_post_install:
+	$(RUN) sed -i 's|$(INSTALL_PREFIX)|$(STAGING_INSTALL_PREFIX)|g' $(STAGING_INSTALL_PREFIX)/lib/pkgconfig/icu-*.pc
```

Comment:
This might break kiwix (libzim)...

Reply:
It does not.
cross/libzim with xapian fails for other reason with updated libicu...

Reply:
indeed, and now fixed.  Still this was an issue captured early on.